rocketpy/mathutils/function.py

```diff
@@ -39,14 +39,33 @@ def __init__(
 
         Parameters
         ----------
-        source : function, scalar, ndarray, string
-            The actual function. If type is function, it will be called for
-            evaluation. If type is int or float, it will be treated as a
-            constant function. If ndarray, its points will be used for
-            interpolation. An ndarray should be as [(x0, y0, z0), (x1, y1, z1),
-            (x2, y2, z2), ...] where x0 and y0 are inputs and z0 is output. If
-            string, imports file named by the string and treats it as csv.
-            The file is converted into ndarray and should not have headers.
+        source : callable, scalar, ndarray, string, or Function
+            The data source to be used for the function:
+
+            - Callable: Called for evaluation with input values. Must have the
+              desired inputs as arguments and return a single output value.
+              Input order is important. Example: Python functions, classes, and
+              methods.
+
+            - int or float: Treated as a constant value function.
+
+            - ndarray: Used for interpolation. Format as [(x0, y0, z0),
+            (x1, y1, z1), ..., (xn, yn, zn)], where 'x' and 'y' are inputs,
+            and 'z' is the output.
+
+            - string: Path to a CSV file. The file is read and converted into an
+            ndarray. The file can optionally contain a single header line.
+
+            - Function: Copies the source of the provided Function object,
+            creating a new Function with adjusted inputs and outputs.
+
+        Notes
```

Comment:
This section should be moved to after line 91.

Reply:
Done in [cd9d10c](https://github.com/RocketPy-Team/RocketPy/pull/485/commits/cd9d10cd4c485582e23ed3629e915e3771a51dd6)